build/setup.d/20-install-packages.sh

```diff
@@ -9,7 +9,7 @@ libyaml-0-2
 linux-image-extra-$(uname -r)
 logentries
 logentries-daemon
-lxc-docker-1.6.2
+lxc-docker
```

Comment:
maybe we should pin the Docker version to 1.7 --- the next Docker version might be broken (for us) again...